packages/toolkit/index.html

```diff
@@ -57,7 +57,7 @@
         USERNAME
       );
 
-      masterApp.CustomStorage.destroy();
+      (await masterApp.CustomStorage).destroy();
```

Comment:
Awaiting a variable seems strange to me, but I understand the why though. Shouldn't App class act as a façade to all those actions?

Reply:
I also find async getters weird, when it started as a getter it was not async, but it evolved.

Maybe we should make it a `getCustomStorage` instead?